org.clementine_player.Clementine.appdata.xml

```diff
@@ -0,0 +1,52 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
Appdata looks good, have you submitted it upstream?

Reply:
I have now :)

Reply:
Link?

Reply:
https://github.com/clementine-player/Clementine/pull/6229